src/build-scripts/build_opencolorio.bash

```diff
@@ -46,9 +61,10 @@ git checkout ${OPENCOLORIO_VERSION} --force
 time cmake -S . -B ${OPENCOLORIO_BUILD_DIR} \
            -DCMAKE_BUILD_TYPE=${OPENCOLORIO_BUILD_TYPE} \
            -DCMAKE_INSTALL_PREFIX=${OPENCOLORIO_INSTALL_DIR} \
-           -DCMAKE_CXX_FLAGS="${OPENCOLORIO_CXX_FLAGS}" \
-           ${OPENCOLORIO_BUILDOPTS}
-time cmake --build ${OPENCOLORIO_BUILD_DIR} --config Release --target install
+           -DCMAKE_CXX_FLAGS_RELEASE="${OPENCOLORIO_CXX_FLAGS}" \
+           -Dyaml-cpp_CXX_FLAGS="${OPENCOLORIO_YAML_CXX_FLAGS}" \
+           ${OPENCOLORIO_BUILDOPTS} ${OPENCOLORIO_SOURCE_DIR}
+time cmake --build . --config Release --target install
```

Comment:
Is there a reason why someone using this utility script should no longer be able to override the build directory name and only allow the build to be in ./build ?

Reply:
Fixed in f2520fa081c68b948da952bb4d19d33eae895f83